src/common/files_js.ml

```diff
@@ -13,10 +13,9 @@
 open Utils
 open Modes_js
 
-let flow_extensions = [
-    ".js"  ;      (* Standard JavaScript files *)
-    ".jsx" ;      (* JavaScript files with JSX *)
-  ]
+let get_flow_extensions root =
```

Comment:
The only reason a `get_options` function is useful is just because it has the boilerplate for unpacking the optional ref (and giving a consistent assert error if somehow some call path attempts to access options before `init` was called).